tests/e2e/tests/lint/lint-with-type-check.ts

```diff
@@ -29,7 +29,6 @@ function check(val: any, fxState: any) {
     fxState.num = val + "";
   }
 }
-
```

Comment:
There was fail `Exceeds the 2 allowed consecutive blank lines` which not fixed with `lint --fix` command.
We can change `no-consecutive-blank-lines` to `3`, if we want to avoid this change.